home/management/commands/translation_tracking.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+from common.translation_utils.update_status import *
```

Comment:
`from ... import *` is generally advised against as it pollutes the namespace.
Only import what you need, e.g. `from common.translation_utils.update_status import `update_status_run`

Reply:
have changed to reference specific functions